shims/spark32/src/main/scala/org/apache/spark/sql/hive/execution/HiveFileFormat.scala

```diff
@@ -101,19 +102,26 @@ class HiveFileFormat(fileSinkConf: FileSinkDesc)
     val outputFormat = fileSinkConf.tableInfo.getOutputFileFormatClassName
     if ("true" == sparkSession.sparkContext.getLocalProperty("isNativeApplicable")) {
       val nativeFormat = sparkSession.sparkContext.getLocalProperty("nativeFormat")
+      val tableProperties = mutable.Map.empty[String, String]
+      tableDesc.getProperties.asScala.foreach {
```

Comment:
It seems `val tableOptions = tableDesc.getProperties.asScala.toMap` is enough.

Besides, please add some suites to test about this.